src/PrestaShopBundle/Bridge/Helper/Listing/HelperBridge/HelperListBridge.php

```diff
@@ -111,24 +111,16 @@ public function generateList(
         if (!($helperListConfiguration->fieldsList && is_array($helperListConfiguration->fieldsList))) {
             return null;
         }
-
-        $this->generateListQuery($helperListConfiguration, $this->context->language->id);
-
         $helper = new HelperList();
 
-        $this->helperListConfigurator->setHelperDisplay($helperListConfiguration, $helper);
-        $helper->_default_pagination = $helperListConfiguration->defaultPagination;
-        $helper->_pagination = $helperListConfiguration->pagination;
-        $helper->tpl_delete_link_vars = $helperListConfiguration->deleteLinksVariableTemplate;
-
-        foreach ($helperListConfiguration->actionsAvailable as $action) {
-            if (!in_array($action, $helperListConfiguration->actions) && isset($helperListConfiguration->$action) && $helperListConfiguration->$action) {
-                $helperListConfiguration->actions[] = $action;
-            }
-        }
-
         /* @phpstan-ignore-next-line */
```

Comment:
This can be removed now?
```suggestion
```

Reply:
it used here https://github.com/PrestaShop/PrestaShop/blob/develop/classes/helper/HelperList.php#L778
though instead of phpstan ignore, I will define the sql property in legacy HelperList and it should be good :+1: